server/server.go

```diff
@@ -74,20 +74,36 @@ func Listen(cfg config.Configuration, publicHandler http.Handler, adminHandler h
 	return
 }
 
+// newAdminServer returns an http.Server with the configured with the AdminPort and
```

Comment:
Nitpick: I think you can get rid of the extra "with the" --> `newAdminServer returns an http.Server with the configured AdminPort and...`